synth_tools/tests/commands/test_agents.py

```diff
@@ -0,0 +1,70 @@
+import json
+from pathlib import Path
+from unittest import mock
+
+import pytest
+from requests import Response
+from typer.testing import CliRunner
+
+from kentik_synth_client import KentikAPIRequestError
+from kentik_synth_client.synth_client import SynthHTTPTransport
+from synth_tools.cli import app
+
+
+def fake_resp(status_code, content) -> Response:
+    resp = Response()
+    resp.status_code = status_code
+    resp._content = content
+    return resp
+
+
+def fake_req(fixture_file):
+    d = Path(__file__).parent / fixture_file
+    with open(d) as stream:
+        return json.load(stream)
+
+
+@pytest.mark.parametrize(
+    "cmd, status_code, expected_in_output, not_expected_in_output, requests",
```

Comment:
It is little bit misleading to call the last field "requests", because it actually contains mocked response payload. We may eventually want to also check that `synth_ctl` is sending correct request.

This may come handy when prepping `synth_tools` for a new version of the API before it is even deployed. The modified version of `synth_tools` may then be used for testing the new version of the API. As a matter of fact, it would come handy right now :-). Han has new version of the API (beta2) ready to test, but he does not have any comprehensive enough test framework. Obviously, `synth_tools` would need to be modified to support new API schema and unit tests would be the only way to test it right now.